korio/src/commonMain/kotlin/com/soywiz/korio/async/AsyncExt.kt

```diff
@@ -115,12 +115,12 @@ class TestCoroutineDispatcher(val frameTime: Int = 16) :
 		}
 	}
 
-	override fun dispatch(context: CoroutineContext, block: Runnable) {
+	fun dispatch(context: CoroutineContext, block: Runnable) {
 		scheduleAfter(0) { block.run() }
 	}
 
-	override fun scheduleResumeAfterDelay(time: Long, unit: TimeUnit, continuation: CancellableContinuation<Unit>) {
-		scheduleAfter(unit.toMillis(time).toInt()) { continuation.resume(Unit) }
+	override fun scheduleResumeAfterDelay(timeMillis: Long, continuation: CancellableContinuation<Unit>): Unit {
```

Comment:
This Long comes from kotlinx.coroutines, so can't be avoided. In Korge uses its own delayer bounded to views that is adjusted to the view speed, so this is not used frequently and shouldn't be a problem.